indico/modules/events/abstracts/forms.py

```diff
@@ -212,7 +212,11 @@ class AbstractJudgmentFormBase(IndicoForm):
     accepted_track = QuerySelectField(_("Track"), [HiddenUnless('judgment', AbstractAction.accept)],
                                       get_label='title', allow_blank=True, blank_text=_("Choose a track..."),
                                       description=_("The abstract will be accepted in this track"))
-    accepted_contrib_type = QuerySelectField(_("Contribution type"), [HiddenUnless('judgment', AbstractAction.accept)],
+    override_contrib_type = BooleanField(_('Override contribution type'),
```

Comment:
We tend to use double quotes for translatable strings, but it's not really enforced, however endorsed!